src/Build.OM.UnitTests/Construction/ElementLocationPublic_Tests.cs

```diff
@@ -82,7 +82,7 @@ public void XmlLocationsAreCached()
         public void LocationStringsMedley()
         {
             string content = @"
-            <Project ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`http://schemas.microsoft.com/developer/msbuild/2003`>
+            <Project ToolsVersion=`msbuilddefaulttoolsversion`>
```

Comment:
You can also remove the ToolsVersion, right?

Reply:
Yup, although I figured this PR was large enough as it was, so probably will follow up with another PR.